### PR TITLE
feat: add gradient effect for sticky header 

### PR DESCRIPTION
**Pull request by unimu-cic**
https://github.com/mermaid-js/zenuml-core/assets/125171539/a38fc8ed-b29a-4e2a-b042-16dcc53541e5

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ module.exports = {
   ],
   theme: {
     extend: {
+      colors: {
```
Comment (MrCoder) [inline tailwind.config.js:19]: What is this for? Should it belong to a different PR?

Comment (unimu-cic) [reply tailwind.config.js:19]: ref: https://tailwindcss.com/docs/gradient-color-stops#customizing-your-theme, the background color of sticky header need gradient color, it's need to be defined in the `tailwind.config.js`

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLayer.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!-- pr-24 to give space for the right most participant.
 TODO: we may need to consider the width of self message on right most participant. -->
 <template>
-  <div class="message-layer pt-24 pb-10">
+  <div class="message-layer relative z-30 pt-24 pb-10">
```
[the remark this answers is not in the document]
Comment (unimu-cic) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLayer.vue:4]: After added the sticky header, the z-index of Lifeline Layer is bigger than z-index of Message  Layer.

[the remark this answers is not in the document]
Comment (unimu-cic) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLayer.vue:4]: > why do we need `relative`?

After added the sticky header, the z-index of Lifeline Layer is bigger than z-index of Message Layer.

File: tailwind.config.js (hunk 1 above)
Comment (unimu-cic) [reply tailwind.config.js:19]: > What is this for? Should it belong to a different PR?

ref: https://tailwindcss.com/docs/gradient-color-stops#customizing-your-theme, the background color of sticky header need gradient color, it's need to be defined in the tailwind.config.js

File: src/components/DiagramFrame/SeqDiagram/LifeLineLayer/LifeLineLayer.vue [hunk 3]
```diff
@@ -1,8 +1,10 @@
 <template>
   <div
-    class="life-line-layer lifeline-layer absolute h-full flex flex-col pt-8 top-0"
-    :style="{ 'min-width': '200px', pointerEvents: renderParticipants ? 'none' : 'all' }"
+    class="life-line-layer w-full lifeline-layer z-30 absolute h-full flex grow flex-col top-0"
+    :style="{ 'min-width': '200px', 'width': `calc(100% - ${leftGap}px)`, pointerEvents: renderParticipants ? 'none' : 'all' }"
```
[the remark this answers is not in the document]
Comment (unimu-cic) [reply src/components/DiagramFrame/SeqDiagram/LifeLineLayer/LifeLineLayer.vue:4]: > Why do we have both `w-full` and `width: calc(100% - ${leftGap}px)`? What is `grow` for?

Removed
